src/three-components/SmoothControls.ts

```diff
@@ -340,21 +390,30 @@ export class SmoothControls extends EventDispatcher {
       maximumRadius
     } = this[$options];
 
-    const {theta, phi, radius} = this[$targetSpherical];
+    const {theta, phi, radius} = this[$destSpherical];
+
+    if (typeof destRadius === 'string') {
+      switch (destRadius) {
+        default:
+        case 'auto':
```

Comment:
The "auto" keyword means apply the ideal camera distance, which I've moved into SmoothControls because I think it is more generically useful than to just <model-viewer>. However, we could tweak the API shape to clean it up a bit; maybe skip the string and use null to represent auto?

Reply:
I put this back.